patterns/2-structured/core-team.md

```diff
@@ -0,0 +1,102 @@
+## Title
+
+Core Team
+
+## Patlet
+
+Is your InnerSource project difficult for people to work with, yet no one is interested in improving it?
```

Comment:
I like the use of a question to open the patlet to invite people who might answer yet to read further. My only comment on this part is the "yet no one is interested in improving it" which seems to imply that even though the IS project is difficult to work with, no one is interested in improving it. It seems to me that this is a causal relationship: because the IS project is difficult to work with, no one is interested in investing the effort to improve it. 
The use of *yet* would need to explore something non-obvious; or else the antecedent would need to focus on why they *should* be interested. E.g., Is your IS project widely needed, yet no one is interested in improving it?

Reply:
I just checked and it looks like we don't have prior instances of a question being used in the Patlet. Maybe because it is a bit harder to express the Problem-Solution combo in a short manner that way?

Anyways, I am all for trying something new :)

Reply:
Updated language, which was easiest for me to express without a question.